src/hooks/useCreditCardInput.tsx

```diff
@@ -187,11 +194,10 @@ function useCreditCardInput() {
 
         props.onChange && props.onChange(e);
 
-        const expiryDateError = getExpiryDateError(
-          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-          //@ts-ignore
-          expiryDateField.current.value
-        );
+        let expiryDateError;
+        typeof expiryDateErrorMessage !== 'undefined'
+          ? (expiryDateError = getExpiryDateError(expiryDateField?.current?.value || '', expiryDateErrorMessage))
```

Comment:
Not sure if nesting conditional accessors `?.` is good practice. My latest push resolved all that.